sif_parser/sif_open.py

```diff
@@ -150,3 +151,119 @@ def xr_open(sif_file, ignore_corrupt=False, lazy=None):
     return xr.DataArray(data, dims=['Time', 'height', 'width'],
                         coords=coords, attrs=new_info)
 
+def np_spool_open(spool_dir, ignore_missing=False, lazy=None):
+    """
+    Read from a directory the binary files and metadata generates via spooling and return a np.array. 
+
+    Parameters
+    ----------
+    spool_dir: 
+        directory path containing the spooling files. 
+        Must contain at least one "sifx_file", one "ini_file" and 
+        one or more "spooled_file(s)":
+        
+    ignore_missing: 
+        True if ignore missing or corrupted *.dat files
+    
+    lazy: either of None | 'memmap' | 'dask'
+        None: load all the data into the memory
+        'memmap': returns np.memmap pointing on the disk
+        'dask': returns dask.Array that consists of np.memmap
+            This requires dask installed into the computer. *Not yet implemented*
+    """
+    dat_files_list = sorted(glob.glob(spool_dir + "/*spool.dat"))
+    ini_file = glob.glob(spool_dir + "/*.ini" )
+    sifx_file = glob.glob(spool_dir + "/*.sifx")
+
+    if len(dat_files_list) < 1:
+        raise ValueError('Not Binary file(s) with extension {} found in the directory provided {} '.format(
+            "*spool.dat", spool_dir))
+    if len(ini_file) < 1:
+        raise ValueError('Not "ini_file" file with extension {} found in the directory provided {} '.format(
+            "*.ini" , spool_dir))
+    if len(sifx_file) < 1:
+        raise ValueError('Not "sifx_file" file with extension {} found in the directory provided {} '.format(
+            "*.sifx", spool_dir))
+
+
+# Checking for missing or corrupted ini file. File must be lenght >=10 and contain the spected keys, vals in the right order.
+    with open(ini_file[0], "r", encoding="utf-8") as f:
```

Comment:
I think we can use [configparser](https://docs.python.org/3/library/configparser.html). If you think it makes the script more readable, this might be a good improvement.